### ISSUE TITLE
Can not preview text file properly

### ISSUE THREAD

**Issue by e190** [kamiyaa/joshuto#162]
The configuration I use directly is   [https://github.com/kamiyaa/joshuto/tree/main/config](url).
But I still can't preview a text file using bat.
I am sure THAT I have installed bat.
![图片](https://user-images.githubusercontent.com/39932163/160739789-f0b58d79-a6b3-49fe-9c07-be15aa024dc9.png)

Can not preview:
![图片](https://user-images.githubusercontent.com/39932163/160739798-24d14b5c-9eb2-42e6-8ae1-70b17ac21f54.png)

**Comment by leontepe** [comment]
Same bug here.

**Comment by kamiyaa** [comment]
I have the ansi color output feature disabled by default right now as `ansi-to-tui` is still unstable.
In order to continue publishing to crates.io, all dependencies must be versioned and not a git repo.
If you want to re-enable this feature, add `syntax_hightlight` to the default features in Cargo.toml:
```toml
[features]
default = [ "devicons", "recycle_bin", "syntax_highlight" ]
```

Hope this helps!

**Comment by alexzanderr** [comment]
If anyone is installing through `cargo`, here's the command:
```shell
cargo install joshuto --features syntax_highlight
```

**Comment by DLFW** [comment]
I think the `ansi-to-tui` problems are solved, right?

Can't we make `syntax_highlight` a default feature again?

**Comment by kamiyaa** [comment]
Closed via b44da4d3c6a4b2764f563ce392afeb7fe9632efc

**Comment by KeqiZeng** [comment]
Facing same issue with the package provided on homebrew(0.9.4). Maybe a new release is required.
